test/make.sh

```diff
@@ -78,13 +84,25 @@ function docker() {
 # This function runs 'terraform validate' and 'terraform fmt'
```

Comment:
the above `docker` function is outdated
You can [remove everything docker-related](https://github.com/terraform-google-modules/terraform-google-memorystore/pull/10#issuecomment-517044237)
cc: @ingwarr

Reply:
will remove it